.github/workflows/weekly-ui-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Weekly UI Test - Enquiry Form
+
+on:
+  #  workflow_dispatch:
+
+  schedule:
+    # Monday at 7 AM UTC - https://cron.help/#0_12_*_*_MON
+    - cron: "0 7 * * MON"
+  #pull_request:
+  #  branches:
+  #    - main
+  workflow_dispatch:
+    inputs:
+      TARGET_URL:
+        type: string
+        description: "Target URL"
```

Comment:
Not done.

Since we are using HOST_URL, this will remain empty and it's not a required field so by default it will run https://ssw.com.au/consulting/angular